howso/direct/core.py

```diff
@@ -458,6 +461,7 @@ def persist(
             "trainee": trainee_id,
             "filename": filename,
             "filepath": filepath,
+            "trainee_id": trainee_id
```

Comment:
Are `trainee` and `trainee_id` always the same value? Seems like they wouldnt be.